numba/pythonapi.py

```diff
@@ -180,6 +180,13 @@ def dict_getitem_string(self, dic, name):
         cstr = self.context.insert_const_string(self.module, name)
         return self.builder.call(fn, [dic, cstr])
 
+    def dict_getitem(self, dic, name):
+        """Returns a borrowed reference
```

Comment:
This docstring doesn't really describe what the function does. How about "Lookup *name* inside dict. Returns a borrowed reference"?